src/compressed_chunk.c

```diff
@@ -76,13 +76,16 @@ static void ensureAddSample(CompressedChunk *chunk, Sample *sample) {
 static void trimChunk(CompressedChunk *chunk) {
     int excess = (chunk->size * BIT - chunk->idx) / BIT;
 
-    assert(excess >= 0); // else we have written beyond allocated memory
+    assert(chunk->size * BIT >= chunk->idx); // else we have written beyond allocated memory
```

Comment:
this is the same as before, just a bit harder to understand. if `chunk->size * BIT` >= `chunk->idx` it the same as : `chunk->size * BIT - chunk->idx >= 0`